### PR TITLE
DSD-677: XXS and XS image size variants

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-677](https://jira.nypl.org/browse/DSD-677)

## This PR does the following:

- adds `xxs` and `xs` image size variants to `Image` and `Card` components
- fixes an image sizing bug for the `row` layout in the `Card` component

## How has this been tested?

- local build of DS

## Accessibility concerns or updates

- existing `a11y` standards were carried over to new image size variants

### Checklist:

- [x] I have updated the Storybook documentation accordingly.
- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.

### Front End Review:

- [x] View [the example in Storybook](https://pr822-sdfske0bj4tym6qnnpivae0fhadsjttd.tugboat.qa/)

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr822-sdfske0bj4tym6qnnpivae0fhadsjttd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c362b87499db6522ff41a2

File: src/components/Card/Card.stories.mdx [hunk 1]
```diff
@@ -221,6 +221,44 @@ prop to override the default width.
 <Canvas>
   <Story name="Card Image Size">
     <SimpleGrid columns={1}>
+      <Card
```
Comment (EdwinGuzman) [inline src/components/Card/Card.stories.mdx:224]: These two would be good to add to the "Row Cards" section as well.

File: src/theme/components/card.ts [hunk 2]
```diff
@@ -137,11 +139,11 @@ const CardImage = {
     const layoutStyles =
       layout === "row"
         ? {
-            display: "flex",
-            flexFlow: {
-              base: "column nowrap",
-              md: "row",
-            },
+            // display: "flex",
```
Comment (EdwinGuzman) [inline src/theme/components/card.ts]: These rules can be removed if they are no longer needed.

File: src/components/Image/Image.stories.mdx [hunk 3]
```diff
@@ -137,55 +172,71 @@ in an element with a fixed width value.
...
-        src="https://placeimg.com/400/300/animals"
-      />
-      <Heading id="twobyone" text="ImageRatios.TwoByOne" />
-      <Image
-        alt="Alt text"
-        imageAspectRatio={ImageRatios.TwoByOne}
-        src="https://placeimg.com/400/300/animals"
-      />
+    <VStack spacing="xl" maxWidth="360px" margin="auto">
+      <div style={{ textAlign: "center", width: "100%" }}>
```
Comment (EdwinGuzman) [inline src/components/Image/Image.stories.mdx]: This doesn't work if you pass `textAlign` and `width` to `VStack`?

File: src/components/Image/Image.stories.mdx [hunk 4]
```diff
@@ -102,28 +104,61 @@ Use the `ImageSizes` prop to set the desired size.
...
-      <Heading id="Large" text="ImageSizes.Large" />
-      <Image
-        alt="Alt text"
-        imageSize={ImageSizes.Large}
-        src="https://placeimg.com/400/300/animals"
-      />
-      <Heading id="Default" text="ImageSizes.Default" />
-      <Image alt="Alt text" src="https://placeimg.com/400/300/animals" />
-    </VStack>
+    <SimpleGrid columns={1}>
```
Comment (EdwinGuzman) [inline src/components/Image/Image.stories.mdx:114]: This is using `SimpleGrid` but the other examples are still using `VStack`. Is this correct? Why not use `SimpleGrid` or the other examples?

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Have some questions.

File: src/components/Card/Card.stories.mdx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Card/Card.stories.mdx:224]: Very true. Not sure how I missed adding those in the first place.

File: src/components/Image/Image.stories.mdx (hunk 3 above)
Comment (bigfishdesign13) [reply src/components/Image/Image.stories.mdx]: `textAlign` did work and I just made that change, but `width` did not.

File: src/components/Image/Image.stories.mdx (hunk 4 above)
Comment (bigfishdesign13) [reply src/components/Image/Image.stories.mdx:114]: I mixed components for styling reasons.  But I cleaned that up and now they are all using `SimpleGrid`.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr822-sdfske0bj4tym6qnnpivae0fhadsjttd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c362b87499db6522ff41a2

**Review by EdwinGuzman — APPROVED** [review]

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr822-sdfske0bj4tym6qnnpivae0fhadsjttd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c362b87499db6522ff41a2

**Pull request merged**